DigitalExcellence/dex-frontend
# Feature/380 redesign homepage

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to the homepage
2. Click on random buttons, do they go to the right place?
3. Check if everything is readable on your screen

## Link to issue

Closes: #380
